mlflow/tensorflow/__init__.py

```diff
@@ -294,6 +282,33 @@ def save_model(
     import tensorflow
     from tensorflow.keras.models import Model as KerasModel
 
+    if signature is None:
+        raise MlflowException(
```

Comment:
Should we add a warning in the docs about this breaking change? A note that specifies something along the lines of "MLflow 2.0 requires that TensorFlow and tf.Keras models are logged with a signature. A signature can be created for logging as shown below:" <brief toy example of generating a signature for a TF model>